src/query/procedure/py_module.cpp

```diff
@@ -1059,8 +1110,8 @@ void CallPythonTransformation(const py::Object &py_cb, mgp_messages *msgs, mgp_g
     py::Object py_graph(MakePyGraph(graph, memory));
     py::Object py_messages(MakePyMessages(msgs, memory));
 
-    utils::OnScopeExit clean_up_graph(PyObjectCleanup(py_graph));
-    utils::OnScopeExit clean_up_messages(PyObjectCleanup(py_messages));
+    utils::OnScopeExit clean_up_graph(PyObjectCleanup(py_graph, true));
+    utils::OnScopeExit clean_up_messages(PyObjectCleanup(py_messages, true));
```

Comment:
It would be better of you declared a constexpr variable and set it to true; that when reading you would know what that constant is doing.
Like kUseBatched = true;

Reply:
Good idea